app/presenters/coronavirus/sub_section_json_presenter.rb

```diff
@@ -3,19 +3,6 @@
 class Coronavirus::SubSectionJsonPresenter
   class MarkdownInvalidError < RuntimeError; end
 
```

Comment:
This exception definition can be removed as it's no longer being raised.

Reply:
done!